src/Infrastructure/TemplateBasedMailer.php

```diff
@@ -36,4 +36,17 @@ public function sendMail( EmailAddress $recipient, array $templateArguments = []
 		);
 	}
 
+	/**
+	 * @throws \RuntimeException
+	 */
+	public function sendMailToOperator( EmailAddress $replyToAddress, array $templateArguments = [] ) {
```

Comment:
IIRC I introduced this TemplateBasedMailer in part to not expose this method to all its users, which generally don't need it.

What made you add it here?